src/screens/SendSheet.js

```diff
@@ -157,17 +156,26 @@ export default function SendSheet(props) {
   const showAssetForm = isValidAddress && !isEmpty(selected);
 
   const isNft = selected?.type === AssetTypes.nft;
-  let color = useColorForAsset({
-    address: selected?.mainnet_address || selected.address,
-  });
+  let colorForAsset = useColorForAsset(
+    {
+      address: selected?.mainnet_address || selected.address,
+    },
+    null,
+    false,
+    true
+  );
   if (isNft) {
-    color = colors.appleBlue;
+    colorForAsset = colors.appleBlue;
   }
 
   const isL2 = useMemo(() => {
     return isL2Network(currentNetwork);
   }, [currentNetwork]);
 
+  const transactionNetwork = useMemo(() => {
+    return isL2Asset(selected?.type) ? selected.type : network;
```

Comment:
this is used in a couple places in this file, i'm just putting a note here that i think it's a bit meh but i am running out of brain cells for today

it is meh bc selected.type is AssetType vs NetworkType which makes this a bit brittle